poetry/core/masonry/builders/wheel.py

```diff
@@ -179,10 +181,23 @@ def _write_metadata(self, wheel):
     def _write_record(self, wheel):
         # Write a record of the files in the wheel
         with self._write_to_zip(wheel, self.dist_info + "/RECORD") as f:
+            record = StringIO()
+            if PY2:
```

Comment:
Is this required since we are getting the compat method to deal with this later on?

Reply:
Yep, the csv writer doesn't like to work well with the StringIO.